### PR TITLE
Add spellings for personal website

### PR DESCRIPTION
**Pull request by jonleung**
This resolves the spelling errors in the personal website workshop: hackclub/hackclub#652

### CODE REVIEW

File: lib/dictionary [hunk 1]
```diff
@@ -359,6 +369,7 @@ Summiter
 Teagan
 TechCrunch
 Technorati
+techy
```
Comment (maxwofford) [inline lib/dictionary:372]: Do we really want to add things like this to our dictionary?

File: lib/dictionary [hunk 2]
```diff
@@ -12,6 +12,7 @@ AIME
 Airbnb
 Airbnb's
 Airtable
+alice
```
Comment (zachlatta) [inline lib/dictionary:15]: Why is this lowercased?

Comment (jonleung) [inline lib/dictionary:15]: because it is part of a username

File: lib/dictionary (hunk 1 above)
Comment (jonleung) [inline lib/dictionary:372]: I am fine with it.

File: lib/dictionary (hunk 2 above)
Comment (jonleung) [inline lib/dictionary:15]: See https://github.com/hackclub/hackclub/pull/652/files#diff-6438b250a154a96449ce4d3382b1d8b1R50

Comment (zachlatta) [inline lib/dictionary:15]: I think the username bit should probably be in an inline code block in that
case (ex. `alice1337`)

On Thu, Jan 21, 2016 at 2:44 PM, Jonathan Leung notifications@github.com
wrote:

> In lib/dictionary
> https://github.com/hackclub/markcop/pull/65#discussion_r50477067:
> 
> > @@ -12,6 +12,7 @@ AIME
> >  Airbnb
> >  Airbnb's
> >  Airtable
> > +alice
> 
> See
> https://github.com/hackclub/hackclub/pull/652/files#diff-6438b250a154a96449ce4d3382b1d8b1R50
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackclub/markcop/pull/65/files#r50477067.

## 

Zach Latta
https://zachlatta.com http://zachlatta.com

**Pull request merged**